src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs

```diff
@@ -268,47 +275,57 @@ private List<string> AddArgs()
             {
                 foreach (var arg in this.VSTestCollect)
                 {
-                    if (arg.Equals("Code Coverage", StringComparison.OrdinalIgnoreCase))
-                    {
-                        isCollectCodeCoverageEnabled = true;
-                    }
+                    isCollectCodeCoverageEnabled = arg.Equals("Code Coverage", StringComparison.OrdinalIgnoreCase);
```

Comment:
>isCollectCodeCoverageEnabled [](start = 20, length = 28)

These variables can be done away with, just invoke below methods, & the string matches

Reply:
Removed the variables